gns/train.py

```diff
@@ -25,14 +25,14 @@
     help='Train model, validation or rollout evaluation.')
 flags.DEFINE_integer('batch_size', 2, help='The batch size.')
 flags.DEFINE_float('noise_std', 6.7e-4, help='The std deviation of the noise.')
-flags.DEFINE_string('data_path', None, help='The dataset directory.')
-flags.DEFINE_string('model_path', 'models/', help=('The path for saving checkpoints of the model.'))
+flags.DEFINE_string('data_path', "/work2/08264/baagee/frontera/gns-mpm-data/gns-data/datasets/WaterDropSample/", help='The dataset directory.')
```

Comment:
```suggestion
flags.DEFINE_string('data_path', None, help='The dataset directory.')
```

Don't hardcode paths